server/channels/app/import_functions.go

```diff
@@ -1428,14 +1435,23 @@ func (a *App) importReplies(rctx request.CTX, data []imports.ReplyImportData, po
 
 		if reply.Id == "" {
 			postsForCreateList = append(postsForCreateList, reply)
+			if replyData.Reactions != nil && len(*replyData.Reactions) > 0 {
+				// although createAt is not unique, I think it is safe to
+				// assume that it could be near-unique especially for the same thread.
+				interimReactionsMap[reply.CreateAt] = replyData.Reactions
```

Comment:
Could you explain what would happen if this assumption didn't hold?

Reply:
Added to the comment itself 👍